src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.hpp

```diff
@@ -133,21 +126,20 @@ class CompositionalMultiphaseFVM : public CompositionalMultiphaseBase
                   CRSMatrixView< real64, globalIndex const > const & localMatrix,
                   arrayView1d< real64 > const & localRhs ) const override;
 
+protected:
 
-  /**
-   * @brief Compute the largest CFL number in the domain
-   * @param dt the time step size
-   * @param domain the domain containing the mesh and fields
-   */
-  void
-  computeCFLNumbers( real64 const & dt, DomainPartition & domain );
-
+  virtual void
+  computeStatistics( real64 const & dt,
+                     DomainPartition & domain ) const override;
```

Comment:
This member functions conditionally computes the statistics if `m_computeStatistics` is `true` (I think that the `doxygen` makes no mention of it).
I would compute the statistics whatever `m_computeStatistics` because it can be clearer.
And I would let the client code check that `m_computeStatistics` is `true` before calling.
If it happens that we have a lot of `if ( m_computeStatistics ) computeStatistics( dt, domain )`, we can make things evolve.

Reply:
I changed my approach to control the output of the statistics. Now they are computed as part of `Tasks`, so we don't have to pollute the solvers with statistics calculations anymore. In particular, this `computeStatistics` function is now gone.